src/Appium.Net/Appium/iOS/IOSDriver.cs

```diff
@@ -227,5 +227,10 @@ public IOSDriver(AppiumLocalService service, DriverOptions driverOptions, TimeSp
         /// </summary>
         /// <returns>The image content of the clipboard as an Image object or null if there is no image on the clipboard</returns>
         public Image GetClipboardImage() => IOSCommandExecutionHelper.GetClipboardImage(this);
+
+        public new void BackgroundApp(int seconds) =>
+            Execute(AppiumDriverCommand.BackgroundApp,
+                AppiumCommandExecutionHelper.PrepareArgument("seconds",
+                    AppiumCommandExecutionHelper.PrepareArgument("timeout", TimeSpan.FromSeconds(seconds).TotalMilliseconds)));
```

Comment:
`timeout` should not be there at all

Reply:
Thanks, I take it this https://github.com/appium/appium-dotnet-driver/issues/309#issuecomment-459757398 is no longer relevant

Reply:
Yeah, previously it was required, but it can drop for now https://github.com/appium/appium-xcuitest-driver/pull/1064
For the backword compatibility, it's better to leave TODO to remove timeout for now if you'd like to keep the timeout :)

Reply:
Ah! thanks for that context. It's an old bug. I'll remove the timeout object to align with the api documentation